pkg/controller/secrets_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package controller
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+	. "github.com/verrazzano/verrazzano-operator/pkg/managed"
```

Comment:
Are we pulling in managed just to get testutils?  If so, we should probably create a testutils package and put the testutils code there.  For example httptest does something like this https://github.com/golang/go/tree/master/src/net/http/httptest

Reply:
I agree but I wanted to handle that in a separate change which I'll do next.

Reply:
ok